lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -848,6 +849,8 @@ def test_check_data_variables(self):
         self.assertEqual(len(failed_var), 4)
         self.assertEqual(set(failed_var), set(['data_variable', 'random_data']))
 
+    @unittest.skipUnless(netCDF4.__netcdf4libversion__ >= '4.3',
+                         'requires netCDF4 library version > 4.3')
```

Comment:
Yep, unittest tells you when a test has been skipped and in verbose mode prints the given message.
I've updated the message to '>= 4.3'